### ISSUE TITLE
A11y_AzureDataStudio_Filter_Color Contrast: Color Contrast ratio for "1 Selected" text is less than 4.5:1

### ISSUE THREAD

**Issue by SaisreeDandamudi** [microsoft/azuredatastudio#15998]
“[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTags**: 
#A11yMAS; #A11yTCS; #SH-AzureDataStudio-June2021; #SQL Azure Data Studio; #Win10; #DesktopApp; #WCAG1.4.3; #AINotInScope; #Color Contrast

### Environment Details:
Azure Data Studio Version: 1.30.0 (user setup)
OS Version: 21H1

### Steps to Reproduce:

1. Launch Azure Data Studio
2. Connect to any SQL server and write a query
3. Tab till results grid and press F3 so that the filters pop-up will open
4. Verify color contrast ratio for "1 selected" text 

### Actual:
Color Contrast ratio for "1 Selected" text is less than 4.5:1

### Expected:
The Color contrast ratio for text content should be >= 4.5:1

### User Impact:
Low vision users will not be able to view the text content if the color contrast ratio is less than 4.5:1

**Comment by Charles-Gagnon** [comment]
@alanrenmsft

**Comment by v-supupp** [comment]
#A11yMAS;#A11yTCS;#SH-AzureDataStudio-June2021;#SQL Azure Data Studio;#Win10; #DesktopApp;#WCAG1.4.3;#AINotInScope;#Color Contrast;#A11ySev2

**Comment by VijayaBharathyA** [comment]
@alanrenmsft 
When we check now after following repro steps we are getting below screen 
![image](https://user-images.githubusercontent.com/100189407/218739019-92b60cbb-8fbc-44ea-adc5-2800fedc1a0b.png)

Could you pls provide repro steps to verify this bug

**Comment by alanrenmsft** [comment]
@VijayaBharathyA you have to first move the keyboard to a cell in the table before pressing F3.

![image](https://user-images.githubusercontent.com/13777222/218912845-d104ab82-b40d-4c99-abaa-ed9f154d46f7.png)

**Comment by VijayaBharathyA** [comment]
#Closed;
Issue is fIxed
![image](https://user-images.githubusercontent.com/100189407/219023389-d5a51813-03f7-4cfc-b4b2-bbae383faa2e.png)
